app/packs/src/apps/mydb/collections/SyncWithMeCollections.js

```diff
@@ -11,7 +11,7 @@ export default class SyncWithMeCollections extends React.Component {
       tree: {
         label: 'Synchronized with me Collections',
         id: -1,
-        children: [{}],
+        children: [],
```

Comment:
What do we do with code duplication like this. Shall we open a refactoring backlog?